src/public/evermore/cipher/index.html

```diff
@@ -160,7 +317,10 @@
 
     function clearInput() {
       document.getElementById("cipherInput").value = "";
-      decodeLetters();
+      //@TODO: Fix clear to remove decoded message string
```

Comment:
TODO?

Reply:
For some reason the clear button deletes the list of all the shifts, but doesn't clear the actual answer and I haven't figured out why yet.